### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
Update the README.md file for basic instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+  This file contains worksheets that are populated with sample data.
+  This file is mainly for demonstration purposes.
+  You can download this file if you would like to see sample entries for the different IAM domains.
+
+- `CH01-ARCHITECT-DESIGN-WORKSHEETS-EXERCISES.xlsx`
+
+  This file contains the worksheets for capturing discovery and design data for each of the IAM Domains.
+  Download this worksheet for the exercises and your own personal use.
+  
+- `99-user.ldif`
```
Comment (ghost) [inline README.md:21]: Oops, this should be `99-user-pretty.ldif`!

**Review by gsdev-ping — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gsdev-ping) [reply README.md:21]: I missed that one. Good catch.

**Review by FRGlover — COMMENTED** [review]
Pleased to see this working out :)

**Review by ramyaparimi — COMMENTED** [review]
Yay!! This is exciting!

Curious question, what is 99-user.ldif?

**Comment by ghost** [comment]
> Yay!! This is exciting!
> 
> Curious question, what is 99-user.ldif?

It is an LDIF file used to extend the LDAP schema in DS. It lets you store custom attributes in your LDAP directory tree.

**Pull request merged**